### PR TITLE
[ZEPPELIN-4132]. Spark Interpreter has issue of SPARK-22393

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?
This PR fix the issue of SPARK-22393 in zeppelin. We can fix this by using `SparkIMain` instead of `IMain`.

### What type of PR is it?
[Bug Fix]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://jira.apache.org/jira/browse/ZEPPELIN-4132

### How should this be tested?
* Unit test is added

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: spark/scala-2.11/src/main/scala/org/apache/zeppelin/spark/SparkScala211Interpreter.scala [hunk 1]
```diff
@@ -43,7 +44,7 @@ class SparkScala211Interpreter(override val conf: SparkConf,
 
   lazy override val LOGGER: Logger = LoggerFactory.getLogger(getClass)
 
-  private var sparkILoop: ILoop = _
+  private var sparkILoop: SparkILoop = _
```
[the remark this answers is not in the document]
Comment (zjffdu) [reply spark/scala-2.11/src/main/scala/org/apache/zeppelin/spark/SparkScala211Interpreter.scala]: Scala210Interpreter already did that

**Comment by zjffdu** [comment]
Merge if no more comment

File: .travis.yml [hunk 2]
```diff
@@ -109,7 +109,7 @@ matrix:
     - sudo: required
       jdk: "oraclejdk8"
       dist: trusty
-      env: BUILD_PLUGINS="true" PYTHON="3" SCALA_VER="2.11" PROFILE="-Pspark-2.2 -Pscala-2.11 -Phadoop2 -Pintegration" SPARKR="true" BUILD_FLAG="install -DskipTests -DskipRat -am" TEST_FLAG="test -DskipRat -am" MODULES="-pl zeppelin-interpreter-integration,zeppelin-web,spark/spark-dependencies" TEST_PROJECTS="-Dtest=ZeppelinSparkClusterTest22,SparkIntegrationTest22,org.apache.zeppelin.spark.* -DfailIfNoTests=false"
+      env: BUILD_PLUGINS="true" PYTHON="3" SCALA_VER="2.11" PROFILE="-Pspark-2.2 -Pscala-2.10 -Phadoop2 -Pintegration" SPARKR="true" BUILD_FLAG="install -DskipTests -DskipRat -am" TEST_FLAG="test -DskipRat -am" MODULES="-pl zeppelin-interpreter-integration,zeppelin-web,spark/spark-dependencies" TEST_PROJECTS="-Dtest=ZeppelinSparkClusterTest22,SparkIntegrationTest22,org.apache.zeppelin.spark.* -DfailIfNoTests=false"
```
Comment (jongyoul) [inline .travis.yml]: May we need to change `SCALA_VER` as well?

**Comment by alonshoham** [comment]
I am trying to upgrade Zeppelin to 0.8.2 and having problems with depInterpreter. I am loading jars with depInterpreter and using them as dependencies in a spark paragraph. The paragraph fails beacause it does not have the jars in its classpath. By setting the "zeppelin.spark.useNew" property to false the spark paragraph succeeds. Could this be related also to this issue resolved in 0.8.2?

**Comment by alonshoham** [comment]
Thanks @zjffdu 
Will the new spark interpreter support depInterprter in the future? I saw that there is an [ignored test](https://github.com/apache/zeppelin/blob/da73634354afcfae7650c88baa7e6c2eae4669c5/spark/interpreter/src/test/java/org/apache/zeppelin/spark/NewSparkInterpreterTest.java#L368) with a comment by you about this behavior. My application relies in the functionality of depInterprter, I create notebooks with code that I compile and load with the depInterpreter. What do you suggest that I do?

**Comment by zjffdu** [comment]
@alonshoham You can use inline configuration instead. 

e.g.
```
%spark.conf

spark.jars           your_local_jar
spark.jars.packages      your_needed_packages
```

**Pull request closed without merging**